brave/brave-core
# Mailerlite query filter 17507

Resolves brave/brave-browser#17507

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open <https://brave.com/?ml_subscriber=17517&ml_subscriber_hash=o3e1>.
2. Confirm that the `ml_subscriber` and `ml_subscriber_hash` parameters have been removed from the URL bar.
